packages/@aws-cdk/aws-apigateway/lib/deployment.ts

```diff
@@ -182,5 +168,7 @@ class LatestDeploymentResource extends CfnDeployment {
 
       this.lazyLogicalId = this.originalLogicalId + md5.digest("hex");
     }
+
+    super.prepare();
```

Comment:
See my note about maybe moving resource dependency calculation to the stack. If you decide to leave Rhiannon here, then add a wee comment on the super call to hint why it Ian needed